metadata/scripts/update_platemap_and_barcodes_with_pathway_info.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
```

Comment:
I found the problem. It's because I renamed the notebook `update_platemap_and_barcodes_with_pathway_info.ipynb` and did not convert it to `.py` using nbconvert in #9. If you look at this change [b1f84db](https://github.com/WayScience/targeted_fibrosis_drug_screen/pull/10/commits/b1f84db76b20d0fb691e83f8038ff03b9db8d962), I just deleted the old file and the new `update_platemap_and_barcodes_with_pathway_info.py` was created after applying the changes to drop the duplicate columns. This was my mistake. Thanks for pointing it out!